emcc.py

```diff
@@ -2883,7 +2883,7 @@ def generate_html(target, options, js_target, target_basename,
         }
       }
       %s
-      var blob = new Blob([code], { type: 'text/javascript' });
+      var blob = new Blob([code], { type: 'application/javascript' });
```

Comment:
does this work in old IE?

Reply:
I don't know. By old IE, which IE version do you mean? I don't unfortunately have a setup to test <IE11. If that is a concern, I could revert this one location back to the old form.

Reply:
I saw stuff like this (see top answer) https://stackoverflow.com/questions/4100872/jquery-code-not-working-in-ie 

Looks like e.g. google.com has `text/javascript` in there, but not `application/javascript` - perhaps for backwards compatibility? see also https://stackoverflow.com/questions/876561/when-serving-javascript-files-is-it-better-to-use-the-application-javascript-or

So I'm not sure what's right here - we need to investigate this if we want to change it. However, changes to stuff related to wasm are probably fine, since that's modern browsers only anyhow - but not sure that helps here.